### ISSUE TITLE
Vivaldi 1.3 middle click paste

### ISSUE THREAD

**Issue by aleskovets** [justdanpo/VivaldiPatches#3]
В версии 1.3 что-то необратимо починили и теперь клик центральной кнопкой мыши приводит ко вставке из буфера. Можно ли это как-нибудь тоже запатчить? Спасибо.

**Comment by justdanpo** [comment]
Клик в какое место? Какая ОС?

**Comment by aleskovets** [comment]
Так, получается довольно специфичное поведение. Я его могу воспроизвести во время редактирования диаграммы в https://www.lucidchart.com: каждый раз во время клика средней кнопкой по панели редактирования вставляется текст из буфера. Использую Linux Mint 17.3 Rosa

**Comment by justdanpo** [comment]
Вообще в браузере уже несколько раз исправляли нежелательную вставку при кликах средней кнопкой.
Когда доберусь до линукса не знаю.
Что должно быть при среднеклике по задумке?

**Comment by aleskovets** [comment]
У них по мид клику идет навигация по панели редактирования: то есть перемещение по рисунку по драгу, как в визио, примерно. Она, кстати, работает. Но при этом из буфера текст тоже одновременно вставляется.
